src/test/java/org/jitsi/meet/test/web/WebParticipant.java

```diff
@@ -56,6 +56,9 @@
             + "&config.gatherStats=true"
             + "&config.disable1On1Mode=true"
             + "&config.analytics.disabled=true"
+            + "&config.disableAudioLevels=true"
```

Comment:
why this change?

Reply:
Oh, that was a miss ... will revert it, sorry.